common/document_parser/lib/responsibility_parse/responsibility_parser.py

```diff
@@ -0,0 +1,484 @@
+from common.document_parser.lib import entities
+from glob import glob
+from tqdm import tqdm
+import json
+import os
+import pandas as pd
+import re
+from nltk.corpus import stopwords
+import string
+
+stopwords = stopwords.words('english')
+# gamechanger-data imports
+from common.document_parser.cli import get_default_logger
+from common.document_parser.lib.document import FieldNames
+
+punctuation_less_period_parentheses = set(string.punctuation).difference({".", "(", ")"})
+
+mid_line_start_numbering_regex = re.compile("^((?!(Table|Figure|Tab\.|Fig\.)\s1\.\s).)*$", flags=re.VERBOSE)
```

Comment:
Since the capture groups aren't used anywhere, I suggest converting them to non-capture groups.
Also, the `VERBOSE` flag isn't needed here.

```
mid_line_start_numbering_regex = re.compile(r"^(?:(?!(?:Table|Figure|Tab\.|Fig\.)\s1\.\s).)*$")
```